query-engine/query-engine-wasm/build.sh

```diff
@@ -13,7 +13,7 @@ OUT_NPM_NAME="@prisma/query-engine-wasm"
 # This little `sed -i` trick below is a hack to publish "@prisma/query-engine-wasm"
 # with the same binding filenames currently expected by the Prisma Client.
 sed -i '' 's/name = "query_engine_wasm"/name = "query_engine"/g' Cargo.toml
-wasm-pack build --release --target $OUT_TARGET
+wasm-pack build --dev --target $OUT_TARGET
```

Comment:
Why `--dev` here?

Reply:
Sorry. should've asked before commiting that. We should definitely go back to `--release` once we get it working, but until then it's conveneint to have readable stacktraces for panics. We can also revert it, i don't have strong feelings about it either way.

Reply:
No need to revert it right now :)
Was just looking for the rationale. Let's keep it this way until we get this to a working situation.